src/Language/PureScript/Sugar/Names/Imports.hs

```diff
@@ -68,6 +73,8 @@ resolveModuleImport env ie (mn, imps) = foldM go ie imps
      -> (SourceSpan, Maybe ImportDeclarationType, Maybe ModuleName)
      -> m Imports
   go ie' (ss, typ, impQual) = do
+    when (mn == C.PrimTypeError) (modify addPrimTypeErrorToEnvironment)
+    when (mn == C.PrimRow) (modify addPrimRowToEnvironment)
```

Comment:
This is wrong, because it adds the entire `Prim.Row` environment to the environment whenever import anything at all from `Prim.Row`. The correct behavior would be to only add the items that are actually imported.

Reply:
This comment appears to have been incorrect, as later tests verified.